### ISSUE TITLE
manifest for OnePlus 11 (kalama/Oxygen OS13.0)

### ISSUE THREAD

**Issue by MhmRdd** [OnePlusOSS/kernel_manifest#2]
I need manifest file to build kernel for OnePlus 11 (kalama) for Oxygen OS 13.0 (Android 13), kernel version 5.14.74-android13.

**Comment by TheWildJames** [comment]
same here

**Comment by OnePlusOSSAdmin** [comment]
We'll provide it with OxygenOS 15.0 first.
For OS 13, we'll update code to 13.1.0.596(EX01), plus the kernel manifest. The kernel version is 5.15.123.

**Comment by TheWildJames** [comment]
thanks!

**Comment by jgamoss** [comment]
hi, can you provide the link for manifest file Oneplus 11 5G,  OOS 13.1  kernel 5.14.74-android13 please ?

**Comment by OnePlusOSSAdmin** [comment]
Sorry for the delay. If everything goes well, we'll provide it next week.

**Comment by fatalcoder524** [comment]
Please update the Kernel Source for android_kernel_common_oneplus_sm8550. 

oneplus/sm8550_t_13.1.0_oneplus11
oneplus_sm8550_t_13.0_oneplus_11
oneplus/sm8550_u_14.0.0_oneplus11

**Comment by OnePlusOSSAdmin** [comment]
It's done for OS14/Android U. For OS13, please wait for some more time.

**Comment by OnePlusOSSAdmin** [comment]
released. https://github.com/OnePlusOSS/kernel_manifest/tree/oneplus/sm8550

**Comment by fatalcoder524** [comment]
branch: oneplus_sm8550_t_13.0_oneplus_11

Android 13 is still missing. Only Android 13.1, 14 and 15 is released.

**Comment by MhmRdd** [comment]
> branch: oneplus_sm8550_t_13.0_oneplus_11
> 
> Android 13 is still missing. Only Android 13.1, 14 and 15 is released.

No, it was previously released when the phone came out, they only stated that they'll update code to 13.1.

**Comment by fatalcoder524** [comment]
> > branch: oneplus_sm8550_t_13.0_oneplus_11
> > Android 13 is still missing. Only Android 13.1, 14 and 15 is released.
> 
> No, it was previously released when the phone came out, they only stated that they'll update code to 13.1.

I'm talking about https://github.com/OnePlusOSS/android_kernel_common_oneplus_sm8550  . I can't see the 13.0 branch here if it was released. Do you mean I have to use 13.1 Kernel in OOS 13?

**Comment by OnePlusOSSAdmin** [comment]
> I'm talking about https://github.com/OnePlusOSS/android_kernel_common_oneplus_sm8550 . I can't see the 13.0 branch here if it was released. Do you mean I have to use 13.1 Kernel in OOS 13?
Yes. What version are you using?

**Comment by MhmRdd** [comment]
> > > branch: oneplus_sm8550_t_13.0_oneplus_11
> > > Android 13 is still missing. Only Android 13.1, 14 and 15 is released.
> > 
> > No, it was previously released when the phone came out, they only stated that they'll update code to 13.1.
> 
> I'm talking about https://github.com/OnePlusOSS/android_kernel_common_oneplus_sm8550  . I can't see the 13.0 branch here if it was released. Do you mean I have to use 13.1 Kernel in OOS 13?

Oh, i don't think the manifest exists, but from the repo where they updated for OOS 15, there are branches for OOS 13.0.

**Comment by fatalcoder524** [comment]
> Oh, i don't think the manifest exists, but from the repo where they updated for OOS 15, there are branches for OOS 13.0.

There are two repos, android_kernel_common_oneplus_sm8550 and android_kernel_oneplus_sm8550. One has and another one doesn't.

**Comment by MhmRdd** [comment]
Probably it was unreleased, which explains why there's no manifest for it.

**Comment by hemasai1109** [comment]
@OnePlusOSSAdmin Can you release manifest for Oneplus Nord 3 mt6983
We didnt get any manifest till now
